augur/tasks/github/messages.py

```diff
@@ -114,22 +114,26 @@ def process_large_issue_and_pr_message_collection(repo_id, repo_git: str, logger
     logger.info(f"{task_name}: Collecting github messages for {len(comment_urls)} prs/issues")
 
     all_data = []
-    for comment_url in comment_urls:
+    skipped_urls = 0
 
+    for comment_url in comment_urls:
         try:
             messages = list(github_data_access.paginate_resource(comment_url))
+            all_data += messages
         except UrlNotFoundException as e:
-            logger.warning(e)
-            continue
-
-        all_data += messages
+            logger.warning(f"{task_name}: 404 on comment URL {comment_url}. Skipping. Reason: {e}")
+            skipped_urls += 1
+        except Exception as e:
+            logger.error(f"{task_name}: Unexpected error on comment URL {comment_url}: {e}", exc_info=True)
```

Comment:
We don't want this because this will potentially obfuscate errors that are happening. We could potentially do this in the future, but right now we don't really look at logs to see the health of collection, we look at flower. So this would hide the error from flower